cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id='viewing-pipeline-logs-using-the-openshift-logging-operator']
+= Viewing pipeline logs using the OpenShift Logging Operator
+
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+
+:context: viewing-pipeline-logs-using-the-openshift-logging-operator
+
+
+toc::[]
+
+
+The logs generated by pipeline runs, task runs, and event listeners are stored in the respective pods. It is often useful to review and analyze such logs for troubleshooting and audits. However, retaining the pods indefinitely leads to unnecessary resource consumption and cluttered namespaces.
+
+To eliminate dependency on pods for viewing pipeline logs, you can use the OpenShift Elasticsearch Operator and the OpenShift Logging Operator provided by {pipelines-title}. These Operators helps you see pipeline logs using the link:https://www.elastic.co/guide/en/kibana/6.8/connect-to-elasticsearch.html[Elasticsearch Kibana] stack, even after deleting the pods containing the logs.
+
+[discrete]
+== Prerequisites
+
+Before trying to view pipeline logs in a Kibana dashboard, ensure the following:
+
+* You are a cluster administrator or have equivalent permissions.
+* You have run pipelines and tasks for a project, and logs are available.
+* You have installed the OpenShift Elasticsearch Operator and the OpenShift Logging Operator.
```

Comment:
This **you are/have** seems superfluous.

a) It doesn't have to be **you** who performed the actions.
b) It's called **Prerequisites**, so it's obvious that it needs to done. We could be adding **you have/need to** in front of every single instruction in the doc.